crates/bevy_ui/src/render/ui.wgsl

```diff
@@ -30,6 +33,10 @@ var sprite_sampler: sampler;
 @fragment
 fn fragment(in: VertexOutput) -> @location(0) vec4<f32> {
     var color = textureSample(sprite_texture, sprite_sampler, in.uv);
-    color = in.color * color;
+    if in.mode == 0u {
```

Comment:
```suggestion
    if in.mode == TEXTURED_QUAD {
```

We should use consts in the shaders too.

Reply:
Yes that make sense